webapp/packages/plugin-data-import/src/DataImportService.ts

```diff
@@ -65,6 +71,11 @@ export class DataImportService {
         processorId,
         file,
         event => {
+          if (isCancelled) {
+            controller.setMessage('plugin_data_import_process_fail_cancel_message');
+            return;
+          }
```

Comment:
probable there will be enough to check `event` object or to just `return` without setting message cuz message will be set by error of cancellation returned by backend. There also can be a case when backend will return success after attempt of cancellation.

I suggest to check if it's possible to determine aborted upload via `event` object. Otherwise do not apply anything from event callback after getting a result (can be fixed in `CustomGraphQLClient.overrideFilesUpload`)